e2e/container/container_test.go

```diff
@@ -59,9 +66,13 @@ func TestContainer(t *testing.T) {
 			// containerd expects it at /sys/fs/cgroup based on
 			// https://github.com/containerd/cgroups/blob/cc78c6c1e32dc5bde018d92999910fdace3cfa27/utils.go#L36
 			tests.Run(&tests.RunOption{BaseOpt: o, CGMode: tests.Hybrid, DefaultHostGatewayIP: hostIP})
-		} else {
+		case "darwin":
 			tests.Run(&tests.RunOption{BaseOpt: o, CGMode: tests.Unified, DefaultHostGatewayIP: "192.168.5.2"})
+		default:
```

Comment:
nit: i am assuming this is for linux but may be good to be explicit, default seems to be a case of unknown